internal/scorecard/helpers/helpers.go

```diff
@@ -17,107 +17,29 @@ package schelpers
-	if len(results) > 0 {
-		name = results[0].Test.GetName()
-		// all results have the same test
-		finalResult.Test = results[0].Test
-		finalResult.MaximumPoints = 1
-		finalResult.EarnedPoints = 1
-	}
-	for _, result := range results {
-		if result.Test.IsCumulative() {
-			return finalResult, fmt.Errorf("cumulative test passed to ResultsPassFail: name (%s)", result.Test.GetName())
-		}
-		if result.Test.GetName() != name {
-			return finalResult, fmt.Errorf("test name mismatch in ResultsPassFail: %s != %s", result.Test.GetName(), name)
-		}
-		if result.EarnedPoints != result.MaximumPoints {
-			finalResult.EarnedPoints = 0
-		}
-		finalResult.Suggestions = append(finalResult.Suggestions, result.Suggestions...)
-		finalResult.Errors = append(finalResult.Errors, result.Errors...)
-		finalResult.Log = result.Log
-	}
-	return finalResult, nil
-}
-
-// ResultsCumulative takes multiple TestResults and returns a single TestResult with MaximumPoints
-// equal to the sum of the MaximumPoints of the input and EarnedPoints as the sum of EarnedPoints
-// of the input
-func ResultsCumulative(results []TestResult) (TestResult, error) {
-	var name string
-	finalResult := TestResult{}
-	if len(results) > 0 {
-		name = results[0].Test.GetName()
-		// all results have the same test
-		finalResult.Test = results[0].Test
-	}
-	for _, result := range results {
-		if !result.Test.IsCumulative() {
-			return finalResult, fmt.Errorf("non-cumulative test passed to ResultsCumulative: name (%s)", result.Test.GetName())
-		}
-		if result.Test.GetName() != name {
-			return finalResult, fmt.Errorf("test name mismatch in ResultsCumulative: %s != %s", result.Test.GetName(), name)
-		}
-		finalResult.EarnedPoints += result.EarnedPoints
-		finalResult.MaximumPoints += result.MaximumPoints
-		finalResult.Suggestions = append(finalResult.Suggestions, result.Suggestions...)
-		finalResult.Errors = append(finalResult.Errors, result.Errors...)
-		finalResult.Log = result.Log
-	}
-	return finalResult, nil
-}
-
-// CalculateResult returns a ScorecardSuiteResult with the state and Tests fields set based on a slice of ScorecardTestResults
-func CalculateResult(tests []scapiv1alpha1.ScorecardTestResult) scapiv1alpha1.ScorecardSuiteResult {
-	scorecardSuiteResult := scapiv1alpha1.ScorecardSuiteResult{}
-	scorecardSuiteResult.Tests = tests
-	scorecardSuiteResult = UpdateSuiteStates(scorecardSuiteResult)
-	return scorecardSuiteResult
-}
-
-// TestSuitesToScorecardOutput takes an array of test suites and generates a v1alpha1 ScorecardOutput object with the
-// provided suites and log
-func TestSuitesToScorecardOutput(suites []TestSuite, log string) scapiv1alpha1.ScorecardOutput {
-	test := scapiv1alpha1.NewScorecardOutput()
+// TestSuitesToScorecardOutput takes an array of test suites and
+// generates a v1alpha2 ScorecardOutput object with the provided suites and log
+func TestSuitesToScorecardOutput(suites []TestSuite, log string) scapiv1alpha2.ScorecardOutput {
```

Comment:
Might be more convenient to make these new helpers methods, ex:
```go
type TestSuites []TestSuite

func (suites TestSuiteSet) ToScorecardOutput(log string) scapiv1alpha2.ScorecardOutput {
	...
}
```

Reply:
if posible I'd like to refactor the helpers stuff when I work on the refactor/or removal of the merge/cumulative logic, It would be in a separate PR.